src/components/Login/Login.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import GoogleLogin from 'react-google-login';
+import * as PropTypes from 'prop-types';
+import { failLogin, successLogin } from '../../services/loginService';
+
+const Login = ({ setAuth }) => {
+  const success = (res) => {
+    successLogin(res);
+    setAuth(true);
+  };
+
+  return (
+    <div style={{ width: '100%', height: '100%', margin: 'auto 50%' }}>
```

Comment:
Try not to use the inline styling, please pass that to a class, so we don't add more work to the refactor we have to do later.